src/test/java/org/fundacionjala/coding/nestor/CustomerTest.java

```diff
@@ -0,0 +1,36 @@
+package org.fundacionjala.coding.nestor;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class CustomerTest {
+    private Customer customer;
+
+    @Test
+    public void testCustomerName() {
+        customer = new Customer("Customer Name");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "Customer Name" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)